### DISCUSSION TITLE
Replantear Hover para Dark Mode

### DISCUSSION THREAD

**Discussion by AnaMontes11** [Telefonica/mistica-design#586] [Proposals]
Actualmente el estado Hover en Dark Mode no funciona.

Esto se debe a que el estado Hover en Dark Mode usa el "backgroundAlternative" (darkModeGrey), que es el mismo que el backgroundContainer (darkModeGrey) por lo que no hay diferencia visual de estados.

![hover](https://user-images.githubusercontent.com/88187691/177508551-24d083a5-583d-4e40-9e72-3ce32967aa2f.png)

**Comment by yceballost** [comment]
alguna idea, sugerencia o solución que tengas en mente?

**Reply by AnaMontes11** [reply]
Una solución podría ser el hover siga usando el backgroundContainer y meterle una capa a 5% de transparencia:

![1](https://user-images.githubusercontent.com/88187691/177749633-61ad03d3-83ab-44af-9473-eac5a9064ab2.png)

Y otra, en vez de usar la transparencia, crearlo como constante por ejemplo llamarle darkModeGreyLow (creo que nos falta un tono intermedio.

![2](https://user-images.githubusercontent.com/88187691/177750345-115e9964-b4ea-49ca-ab6b-0fd717a65f74.png)

**Reply by yceballost** [reply]
y en light mode? creo que lo que propones es lo que se haría con lo que comenta debajo @aweell

**Reply by aweell** [reply]
Justo la propuesta sería algo así:

https://codepen.io/AlexBueno/pen/OJvRLEX

**Reply by AnaMontes11** [reply]
Eso es!

**Comment by aweell** [comment]
Una solución es que estos estados tengan su token propio (hover, pressed, selected) y no reutilicen tokens que aplican a elementos no interactivos también.

En este caso propondría por ejemplo un token `containerBackgroundHover` siguiendo la línea de otros ya existentes como `buttonPrimaryBackgroundHover`.

**Comment by yceballost** [comment]
Añado un caso más para tener los hover diferenciados. Al usar actualmente los backgroundAlternative, en el caso de blau y los hover de las listas funciona un poco mal, y si tuviese un backgroundAlternative cerca yo creo que se vería raro
https://ibit.ly/Z3Rg

**Comment by yceballost** [comment]
Se va a trabajar en la tarea → https://github.com/Telefonica/mistica-design/issues/677
